starter-dev-backend/package.json

```
@@ -13,11 +13,17 @@
     "cors": "^2.8.5",
     "crypto": "^1.0.1",
     "express": "^4.17.1",
+    "express-graphql": "^0.12.0",
+    "graphql": "^16.3.0",
+    "graphql-scalars": "^1.15.0",
+    "graphql-tag": "^2.12.6",
+    "graphql-tools": "^8.2.0",
```

You should remove packages we ended up not using. 😄 
```suggestion
    "express-graphql": "^0.12.0",
    "graphql": "^16.3.0",
```